test/addons/argocd/test

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import time
+
+from drenv import kubectl
+from drenv import commands
+
+
+def wait(cluster):
+    print("Waiting until all deployments are available")
+    kubectl.wait(
+        "deploy",
+        "--all",
+        "--for=condition=Available",
+        "--namespace=argo-test",
+        "--timeout=300s",
+        context=cluster,
+    )
+
+
+if len(sys.argv) != 4:
+    print(f"Usage: {sys.argv[0]} hubcluster drcluster1 drcluster2")
+    sys.exit(1)
+
+cluster = sys.argv[1]
+
+commands.run("argocd", "app", "create", "guestbook", "--repo",
+             "https://github.com/argoproj/argocd-example-apps.git",
+             "--path", "guestbook", "--dest-server", "https://kubernetes.default.svc",
+             "--dest-namespace", "argo-test", "--sync-option", "CreateNamespace=true",
+             "--sync-policy", "automated", "--kube-context", cluster)
+
+time.sleep(10)
+wait(cluster)
```

Comment:
Why do we wait for the cluster again? maybe you want to wait for the app?

Reply:
already changed code